CommentingPractice/test/sumTest.js

```diff
@@ -0,0 +1,36 @@
+const assert = require('chai').assert; 
+const Services = require('./allServices'); 
+
+describe('Sum',function() {
+    it('should return the sum of two positive numbers',function() {
+        let x = 6;
+        let y = 2;
+        let result = Services.Sum.sum(x,y);
+        assert.equal(result,6+2);
+    });
+    it('should return a negative number', function(){
+        let x = 2;
+        let y = -3;
+        let result = Services.Sum.sum(x,y);
+        assert.equal(result, -1);
+    });
+    it('should return a zero as a valid number', function(){
+        let x = 3;
+        let y = -3;
+        let result = Services.Sum.sum(x,y);
+        assert.equal(result, 0);
+    });
+    it('should return numeric only', function(){
+        let x = 2;
+        let y = 6;
+        let result = Services.Sum.sum(x,y);
+        assert.typeOf(result, 'number');
+    });
+    it('should return proper text when user provide non-numeric value', function(){
+        let x = 7;
+        let y = "@";
+        let result = Services.Sum.sum(x,y);
+        assert.equal(result, "Please provide Numbers only")
```

Comment:
The only problem here is if the message in the sum method changes this test will fail in the future. One possible solution would be changing what we're testing for. Another would be change the sum service such that it exposes a property representing the error message then validate against that property.